validations/src/validations.rs

```diff
@@ -299,6 +416,12 @@ pub fn validate_data_request_output(request: &DataRequestOutput) -> Result<(), T
         return Err(TransactionError::InsufficientWitnesses);
     }
 
+    if !((51..100).contains(&request.min_consensus_percentage)) {
```

Comment:
How does this compare to good old comparison, performance-wise?

Reply:
![Captura de pantalla de 2020-01-08 15-38-35](https://user-images.githubusercontent.com/44604217/71986564-f9da3b00-322c-11ea-9ae2-a8cc400bcc62.png)

Reply:
AWESOME